src/Form/MetadataDisplayForm.php

```diff
@@ -397,25 +481,31 @@ public static function ajaxPreview($form, FormStateInterface $form_state) {
             '#markup' => $message,
           ]
         ];
-        $output['twig_vars'] = [
-          '#type' => 'details',
-          '#open' => FALSE,
-          '#title' => 'Twig Variables',
-          'render' => [
-            'table' => $var_table
-          ],
-        ];
-        $output['json_unused'] = [
-          '#type' => 'details',
-          '#open' => FALSE,
-          '#title' => 'Unused JSON keys',
-          'render' => [
-            'table' => $json_table
-          ],
-        ];
+        if (!empty($json_table)) {
```

Comment:
Don't understand this condition here. Why are you checking for $json_table? and why double $output['json_unused']. Maybe I am missing something?

Reply:
Hmm, I guess this doesn't make sense in the Exception since it means the try didn't work. Will remove.

Reply:
Resolved via 029b2caa68c59d9a55de25fb7547ff8c32110b20.